cmd/gitserver/gitserver.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/openshift/origin/pkg/cmd/infra/gitserver"
+	"github.com/openshift/origin/pkg/cmd/util/serviceability"
+)
+
+func main() {
+	defer serviceability.BehaviorOnPanic(os.Getenv("OPENSHIFT_ON_PANIC"))()
+	defer serviceability.Profile(os.Getenv("OPENSHIFT_PROFILE")).Stop()
+
+	if len(os.Getenv("GOMAXPROCS")) == 0 {
+		runtime.GOMAXPROCS(runtime.NumCPU())
+	}
+
+	basename := filepath.Base(os.Args[0])
+	command := gitserver.NewCommandGitServer(basename)
```

Comment:
`cli.CommandFor` here and `gitserver.NewCommandGitServer` in the body of `CommandFor`?